docs/development/CUSTOM_RULES.md

```diff
@@ -46,27 +46,30 @@ export const rule = createRule({
   create(context) {
     return {
       FunctionDeclaration(node) {
-        if (/^[a-z]/.test(node.id.name)) {
+   if(node.id!=null){
```

Comment:
This code isn't formatted very nicely, and [Prettier](https://prettier.io/) is complaining as part of our build checks. That's why this PR has a big red :x: on its status. https://github.com/typescript-eslint/typescript-eslint/runs/5741983136?check_suite_focus=true

I'd recommend opening the repository in VS Code -either by cloning locally or with the GitHub online editor and using the [Prettier VS Code extension](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) to format it.

Alternately, you can use `prettier --write` on the command-line.

Reply:
Ok got it